packages/docs/pages/reference/elements/table.mdx

```diff
@@ -8,12 +8,170 @@ import {
 
 # `<table>`
 
-Short description of what the `<table>` tag does.
+The `<table>` tag produces a container for the [`<tabular>`](./tabular) tag and assigns to it a number which can be associated with a title for table. The [`<tabular>`](./tabular) tag produces what one normally thinks of as a table
+consisting of rows and columns.  For complex tables, there is an [online table
+builder](https://www.latex-tables.com/) which can generate PreTeXt output.
 
 ## Syntax
 
 <SyntaxDisplay name="table" />
 
 ## Examples
 
-    
\ No newline at end of file
+### A comment about table borders
+
+Placing borders in tables exactly where you want them requires that you
+understand in which elements the attributes `top`, `left`, `bottom`, and
+`right` may be applied. Note that while horizontal borders (rules)
+ are easy, vertical rules have been made intentionally more work "to gently
+ guide authors towards good choices" in document design.
+[Table 4.18.3]
+(https://pretextbook.org/doc/guide/html/topic-tabular.html#topic-tabular-13-7)
+is the definitive source.
+
+
+### A simple table
+
+ In this example, all cell entries are right-justified; the table has a header
+ row and some cells have borders.
+
+  ```ptx-example
+<FRAGMENT parents="subsection section">
+    <table>
+          <title>Sample check registry</title>
+          <tabular halign="right">
```

Comment:
This should be the first example in `<tabular>`. It's simple enough to be understood!